app/qml/components/MMListDrawer.qml

```diff
@@ -30,7 +30,7 @@ MMDrawer {
       width: parent.width
       height: Math.min( root.drawerContentAvailableHeight, contentHeight )
 
-      enabled: contentHeight > height
+      enabled: emptyStateDelegateLoader.visible
```

Comment:
That condition was important - you would be scrolling otherwise even when you do not need to (content fits the screen)
```suggestion
      enabled: contentHeight > height || emptyStateDelegateLoader.visible
```

Reply:
Enabled = false blocks all events (like mouse click) on all children

Reply:
Which is exactly source of reported bug

Reply:
That's fine, so it could be like:
```qml
enabled: emptyStateDelegateLoader.visible ? emptyStateDelegateLoader.visible : contentHeight > height
```

Reply:
scrollbar is only for no-items element, so if emptyStateDelegate is not shown whole scrollbar should be off (which is the fix in the PR?)